typescript-client-sdk/client.ts

```diff
@@ -36,32 +36,30 @@ export class HathoraClient {
     return res.token;
   }
 
-  public async createPrivateLobby(token: string): Promise<string> {
-    return await this.postJson(
-      `https://api.hathora.dev/lobby/v1/${this.appId}/create/private?local=${
-        this.localConnectionDetails === undefined ? "false" : "true"
-      }`,
-      {},
+  public async createPrivateLobby(token: string, { region = "Washington_DC", initialConfig = "{}" } = {}): Promise<string> {
```

Comment:
should `initialConfig` here be `{}` instead of `"{}"`?

Reply:
I suppose so! Didn't know fetch could accept an object ([before](https://github.com/hathora/buildkits/assets/1247668/ab5e8ac5-d6cd-4ff6-9409-77b9e9bbba32), [after](https://github.com/hathora/buildkits/assets/1247668/1b916ad5-33fd-402f-bf30-5112589f734f)).